### ISSUE TITLE
Error: Line number greater than maximum line

### ISSUE THREAD

**Issue by b-filip** [AtomLinter/linter-elixirc#63]
```
Error: Line number (8) greater than maximum line (6)
    at Object.rangeFromLineNumber (/home/crabman/.atom/packages/linter-elixirc/node_modules/atom-linter/lib/index.js:179:11)
    at parseError (/home/crabman/.atom/packages/linter-elixirc/lib/init.coffee:105:28)
    at /home/crabman/.atom/packages/linter-elixirc/lib/init.coffee:130:22
```

![2016-04-27 01-05-01](https://cloud.githubusercontent.com/assets/6838251/14835751/1da80d84-0c14-11e6-888b-49b3ad9d9c58.png)

This happens when I press Ctrl+S to save in this file.

**Comment by Arcanemagus** [comment]
If you run the linter manually on the file in the CLI do you see it reporting an error message on line 8?

**Comment by b-filip** [comment]
@Arcanemagus how do I run it?

**Comment by Arcanemagus** [comment]
Something along these lines:

```shell
elixirc --ignore-module-conflict --app mix --app ex_unit -o /tmp test/tests.exs
```

To find out _exactly_ how it is being ran you would need to set a breakpoint [here](https://github.com/AtomLinter/linter-elixirc/blob/v1.1.7/lib/init.coffee#L162) and [here](https://github.com/AtomLinter/linter-elixirc/blob/v1.1.7/lib/init.coffee#L166), and then if the breakpoint on line 162 was triggered, check the contents of `elixircArgs`.

**Comment by jessejanderson** [comment]
Having the same issue. :(

**Comment by Arcanemagus** [comment]
@jessejanderson Then the same questions apply to you :stuck_out_tongue:.

Can you file a new issue though? It's unlikely you are seeing this for the same reason @b-filip is.

**Comment by bbugh** [comment]
This happens in several places but it is most easily reproduced with `.exguard.exs`. I made a very basic sample repository for you to reproduce the issue. It works 100% of the time.

https://github.com/bbugh/linter-elixirc-issue-63

![image](https://cloud.githubusercontent.com/assets/438465/18033716/7b10f2cc-6cf1-11e6-9130-d02e6b7a12a9.png)

This has also happened on files that don't end in `.exs`, but I can't figure out those cases. For example, it happened once was when I was messing around with namespacing  my modules in an `.ex` file (like `Namespace.Module`).

**Comment by remiq** [comment]
@bbugh Version 1.1.11 suppresses this message.
